commit cdc85e9e7a14932808471f823bc974031a0c4842
Author: Allen Winter <allen.winter@kdab.com>
Date:   Mon Jan 25 14:57:20 2016 -0500

    libical.pc.in - set the version to the full major.minor.patch
    as requested by Milan.

diff --git a/libical.pc.in b/libical.pc.in
index 7774dce2..5c2debd8 100644
--- a/libical.pc.in
+++ b/libical.pc.in
@@ -7,6 +7,6 @@ iculib=@ICU_LIBRARIES@ @ICU_I18N_LIBRARIES@
 
 Name: libical
 Description: An implementation of basic iCAL protocols
-Version: @VERSION@
+Version: @LIBICAL_LIB_VERSION_STRING@
 Libs: -L${libdir} -lical -licalss -licalvcal ${threadslib} ${iculib}
 Cflags: -I${includedir}
